XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs

```diff
@@ -46,30 +49,29 @@ public string EmptyViewText
 
         public async Task OnAppearing()
         {
-            if (Contributors != null)
+            if (Contributors.Any())
                 return;
 
             try
             {
                 var contributors = await gitHubClient.Repository.GetAllContributors("xamarin", "XamarinCommunityToolkit");
-
-                if (contributors is null)
-                    return;
-
                 //Initiate poor mans randomizer for lists
                 //Note: there are better options for real production worthy large lists : https://stackoverflow.com/questions/273313/randomize-a-listt
                 //But for now this linq version will do
-                var rnd = new Random();
-                Contributors = new ObservableCollection<RepositoryContributor>(contributors.Select(x => new { value = x, order = rnd.Next() }).OrderBy(x => x.order).Select(x => x.value));
-            }
-            catch(Exception ex)
-            {
+                var random = new Random();
+                var result = contributors?.OrderBy(x => random.Next()).ToArray();
+                if (result != null)
+                    Contributors = result;
             }
-            finally
+            catch
             {
-                if(Contributors is null || !Contributors.Any())
-                    EmptyViewText = "No data loaded...";
+                // Suppress
             }
+
+            if (Contributors.Any())
+                return;
+
+            EmptyViewText = "No data loaded...";
```

Comment:
Why don't we always set the `EmptyViewText`? The empty view is only shown whenever the collection is empty, right? We should not have to think about that logic. If we do, then we should consider if it's a bug in Forms

Reply:
```string emptyViewText = "Loading data...";```

Initially, EmptyViewText is used for showing a different message. So here we replace it :)